### PR TITLE
Cleanup columns

### PR DESCRIPTION
**Pull request by szecket**
new look to create some space for labels and content.

### CODE REVIEW

File: tools/lsp/ui/views/property-view.slint [hunk 1]
```diff
@@ -49,29 +49,31 @@ component ResetButton inherits Button {
 component NameLabel inherits HorizontalLayout {
     in property <ElementInformation> element-information;
     in property <PropertyInformation> property-information;
-
+    in property <bool> debug;
```
Comment (hunger) [inline tools/lsp/ui/views/property-view.slint:52]: Please remove the debug code

File: tools/lsp/ui/views/property-view.slint [hunk 2]
```diff
@@ -49,29 +49,31 @@ component ResetButton inherits Button {
...
-
     BodyText {
         min-width: EditorSizeSettings.min-prefix-text-width;
-        min-height: 2rem;
 
         text: root.property-information.name;
 
         font-size: 1rem;
         font-weight: property-information.value.code != "" ? EditorFontSettings.bold-font-weight : EditorFontSettings.light-font-weight;
+        font-italic: property-information.value.code != "" ? false : true;
```
Comment (hunger) [inline tools/lsp/ui/views/property-view.slint:69]: This is `property-information.value.code == ""`

**Review by hunger — APPROVED** [review]

**Comment by NigelBreslaw** [comment]
https://github.com/user-attachments/assets/9e53ea9d-5ede-49dc-8b60-c17aeb7decd4

Something odd is going on. Does it normally take 2 clicks to focus the input areas? Then the flashing cursor is on the left side of the number, not the right.

**Comment by hunger** [comment]
Is the ComboBox for enums intentionally shorter than the rest?

**Comment by hunger** [comment]
@NigelBreslaw: This PR does not make a difference there though, the current implementation I have also needs two clicks to switch :-(

File: tools/lsp/ui/views/property-view.slint (hunk 2 above)
Comment (szecket) [reply tools/lsp/ui/views/property-view.slint:69]: I reversed it so the true/false is also reverse.  I think that's what you meant?

**Pull request merged**